api/pom.xml

```diff
@@ -17,49 +17,25 @@
 
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
         <version>1.0.6</version>
+        <relativePath />
     </parent>
 
-    <modelVersion>4.0.0</modelVersion>
     <groupId>jakarta.servlet.jsp.jstl</groupId>
     <artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
-    <packaging>jar</packaging>
     <version>2.0.0-SNAPSHOT</version>
-    <name>JavaServer Pages(TM) Standard Tag Library API</name>
+    <packaging>jar</packaging>
 
-    <description>JavaServer Pages(TM) Standard Tag Library API</description>
+    <name>Jakarta Server Pages Standard Tag Library API</name>
```

Comment:
Can we just make this "Jakarta Standard Tag Library API"?

Reply:
Yes, done!